alert.html

```diff
@@ -0,0 +1,44 @@
+<link rel="import" href="../polymer/polymer.html">
+<link rel="import" href="../vui-colors/colors.html">
+
+<dom-module id="vui-alert">
+    <template>
+        <style>
+            :root {
+                --very-red: #ff7962;
+                --less-red: #ffeeec;
+                --border-width: 5px;
+                --border-radius: 3px;
```

Comment:
In most cases we express dimensions in `rem` units.  So, based on a base font-size of `20px`, this the border-radius becomes `0.3rem`.  This approach is however not recommended for single pixel borders because some browser handle partial, sub-1px values poorly.